types/ahocorasick/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for ahocorasick 1.0.2
+// Project: https://github.com/BrunoRB/ahocorasick
+
+declare class AhoCorasick {
+    constructor(keywords: string[]);
+
+    search(match: string): Array<[number, string[]]>;
+}
+
+export = AhoCorasick
```

Comment:
```suggestion
export = AhoCorasick;
export as namespace AhoCorasick;
```

Reply:
For more details https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module

Reply:
Completed

Reply:
After adding this modification, the access control will report an error